napari/layers/labels/labels.py

```diff
@@ -718,7 +718,7 @@ def show_selected_label(self, filter_val):
         self._show_selected_label = filter_val
         self.refresh()
 
-    @Layer.mode.getter
+    @property
     def mode(self):
```

Comment:
```suggestion
    def mode(self) -> str:
```
(since we are fixing typing after all!)